internal/cmd/update.go

```diff
@@ -50,6 +52,7 @@ func init() {
 	updateCmd.Flags().BoolVarP(&updateFlags.commit, "commit", "c", false, "Automatically commits the change.")
 	updateCmd.Flags().StringVarP(&updateFlags.commitMessage, "commit-message", "m", "", "Use a custom message for the commit.")
 	updateCmd.Flags().BoolVar(&updateFlags.cloud, "cloud", false, "Use MACH composer cloud to check for updates.")
+	updateCmd.Flags().BoolVar(&updateFlags.gitFallback, "git-fallback", false, "Fallback to git if composer cloud check fails.")
```

Comment:
I think we want to default to true here? That way we don't change prior behaviour? I think if we want to override that logic we can always set `--git-fallback=false`

Reply:
Agreed, it would be neater to not have breaking changes, altho when discussing this we figured most project probably don't rely on this behaviour and is actually unwanted, but this we don't know 100% sure.
What do you prefer? Ask around / small poll and introduce different behaviour or default to `true`?

Reply:
Lets just merge this, and when people come to complain we can tell them which flag to set to get back the old system :+1: